docs/README.md

```diff
@@ -52,6 +48,21 @@ To configure the CLI to use the guest account, you can run the following command
 
 To verify your CLI setup, try [creating and running an action](#openwhisk-hello-world-example).
 
+## Setting up the depreicated Python version of the CLI
```

Comment:
typo in "depreicated"
I would change the title to "Setting up the deprecated OpenWhisk CLI (Python based)"

I guess the section would live here for a temporary time until it get's completely removed.